variables.tf

```diff
@@ -1,3 +1,7 @@
+########################
+#### Variable values ###
```

Comment:
variable values does not means anything to anyone reading this, better to do not have any comment than obvious one

Reply:
Fixed in next PR.